### ISSUE TITLE
Casting works only once then not again

### ISSUE THREAD

**Issue by rhetsigga** [yuliskov/SmartTubeNext#1175]
Pairs and cast the first time no issue. Then every time after (TV has been turned off) it doesn't connect. It doesn't appear on the list of devices (only the regular Youtube does). And when I try add it again, it will make a connection but won't connect - The cast icon sort of animates continuously.

TCL 4-series 2021 Google TV - Android 11

**Comment by AlexeSimon** [comment]
Might be a duplicate of https://github.com/yuliskov/SmartTubeNext/issues/1066
Maybe try the temporary solutions mentioned there.

**Comment by rhetsigga** [comment]
Yes and also later today I will roll back to an older version maybe Nov 2021 or before to see.

**Comment by Omp26** [comment]
Same for me.
